standalone/src/models/product-type/attribute-definition/attribute-definition-draft/fields-config.ts

```diff
@@ -52,6 +52,7 @@ export const restFieldsConfig: TModelFieldsConfig<TAttributeDefinitionDraftRest>
           AttributeEnumTypeDraftRest.random(),
         ])
       ),
+      level: oneOf(...Object.values(levels)),
```

Comment:
Based on the conversation I had with @jaikumar-tj when I migrated this model, the `level` field should have the `levels.Variant` value by default.